libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -63,30 +64,21 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
     });
 	}
 	
-	private getTotalPages(allIds: number[], pageSize: number) {
-		return Math.floor(allIds.length / pageSize) + 1;
+	private getTotalPages(allIds: string[], pageSize: number) {
+		return allIds.length % pageSize ? Math.floor(allIds.length / pageSize) + 1: allIds.length/pageSize;
```

Comment:
This can just be replaced with `Math.ceiling(allIds.length/pageSize);`

Reply:
done